### PR TITLE
Replace GitHub pages deployment with manual deployment

### PR DESCRIPTION
**Pull request by Fryguy**
@bdunne Please review.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd dest
+
+echo "www.manageiq.org" > CNAME
+
+git config --global user.name "ManageIQ Bot"
+git config --global user.email "noreply@manageiq.org"
+
+git clone --bare --branch master --depth 5 https://miq-bot:${GITHUB_TOKEN}@github.com/ManageIQ/manageiq.github.io .git
```
Comment (bdunne) [inline deploy.sh]: Is there a space before the `.git`?

Comment (Fryguy) [reply deploy.sh]: Yes...that is separating the URL from the name of the target dir (which is `.git`)

Comment (Fryguy) [reply deploy.sh]: Put another way...this could have also been: `https://.../manageiq.github.io.git .git`

Comment (bdunne) [reply deploy.sh]: Oh, I see, sorry for the confusion.  I was thinking that was supposed to be part of the URL

**Pull request merged**